### DISCUSSION TITLE
Support "Save and continue editing" for CK Editor configs

### DISCUSSION THREAD

**Discussion by MoritzLost** [craftcms/ckeditor#108] [Ideas] [answered]
I'm migrating a site from Redactor to CK Editor. In the process, I have to fiddle with the CK Editors configs a lot. It's really inconvenient that `CMD + S` goes back to the configuration list after saving, instead of staying in the configuration. Would it be possible to have `CMD + S` stay on the current screen, allowing me to quickly try out multiple changes to the config? So the shortcut would work the same way it does on entry screens (_Save and continue editing_).

**Comment by brandonkelly** [comment, marked as answer]
Added for the next release! (54a351d82dfac9ddde2d71ce6a46c1120f1128f4)

**Reply by MoritzLost** [reply]
Awesome, thanks @brandonkelly!

**Reply by brandonkelly** [reply]
CKEditor 3.5.0 is out with this :tada:

Note that the update requires [Craft 4.5.0](https://craftcms.com/blog/craft-4-5-released).
